lib/rouge/lexers/reasonml.rb

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    load_lexer 'ocaml/common.rb'
+
+    class ReasonML < OCamlCommon
+      title "ReasonML"
+      desc 'New syntax on top of OCaml ecosystem (reasonml.github.io)'
+      tag 'reasonml'
+      filenames '*.re', '*.rei'
+      mimetypes 'text/x-reasonml'
+
+      def self.keywords
+        @keywords ||= super + Set.new(%w(
+          switch
+        ))
+      end
+
+      state :root do
+        rule %r/\s+/m, Text
+        rule %r/false|true|[(][)]|\[\]/, Name::Builtin::Pseudo
```

Comment:
This would match `()`. Is this the empty list? And similarly is '[]' an array-like object?

Reply:
yes `()` is a special object